custom_components/bodymiscale/translations/it.json

```diff
@@ -0,0 +1,48 @@
+{
+  "config": {
+    "error": {
+      "already_configured": "Già configurato",
+      "height_limit": "Altezza troppo elevata (limite: 220 cm)",
+      "invalid_date": "Data non valida"
+    },
+    "step": {
+      "options": {
+        "data": {
+          "height": "Altezza",
+          "impedance": "Sensore impedenza",
+          "weight": "Sensore peso"
+        },
+        "data_description": {
+          "impedance": "Se la tua bilancia non fornisce il valore di impedenza lascia questo campo vuoto."
+        }
+      },
+      "user": {
+        "data": {
+          "birthday": "Data di nascita",
+          "gender": "Genere",
+          "name": "Nome"
+        }
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "height": "Altezza",
+          "impedance": "Sensore impedenza",
+          "weight": "Sensore peso"
+        },
+        "data_description": {
+          "impedance": "Se la tua bilancia non fornisce il valore di impedenza lascia questo campo vuoto."
+        }
+      }
+    }
+  },
+  "state": {
+    "_": {
+      "ok": "OK",
```

Comment:
```suggestion
```

Please remove this line, as it is the same in English (`en.json`).
As English is the fallback it will be automatically used and we save space :)